tests/beta/tapes/test_rev.py

```diff
@@ -17,20 +17,186 @@
 
 import pennylane as qml
 from pennylane.beta.interfaces.autograd import AutogradInterface
-from pennylane.beta.tapes import ReversibleTape, QNode
+from pennylane.beta.tapes import QuantumTape, ReversibleTape, QNode, qnode
 from pennylane.beta.queuing import expval, var, sample, probs, MeasurementProcess
 
 
 thetas = np.linspace(-2 * np.pi, 2 * np.pi, 8)
 
 
-class TestExpectationJacobian:
+class TestReversibleTape:
+    """Unit tests for the reversible tape"""
+
+    def test_diff_circuit_construction(self, mocker):
+        """Test that the diff circuit is correctly constructed"""

```

Comment:
what is the diff circuit? :thinking:

Reply:
Oh, this is the variable name you give to the circuit `between_ops.inv() + generator + between_ops` 😆 I didn't know what else to call it, and thought the variable name was good enough.

Any better suggestions?